Code/MinimalLib/minilib.h

```diff
@@ -160,21 +162,62 @@ class JSMol {
       unsigned int maxCutBonds) const;
 #endif
 
-  std::unique_ptr<RDKit::RWMol> d_mol;
   static constexpr int d_defaultWidth = 250;
   static constexpr int d_defaultHeight = 200;
+
+ protected:
+  JSMolBase() = default;
+};
+
+class JSMol : public JSMolBase {
+ public:
+  JSMol() : d_mol(new RDKit::RWMol()) {}
+  JSMol(RDKit::RWMol *mol) : d_mol(mol) { checkNotNull(); }
+  JSMol(const JSMol &other) { d_mol.reset(new RDKit::RWMol(other.get())); }
+  JSMol &operator=(const JSMol &other) {
+    d_mol.reset(new RDKit::RWMol(other.get()));
+    return *this;
+  }
+  RDKit::RWMol &get() const {
```

Comment:
I think this should not be `const` since it returns a non-const reference to a data member.

Probably you need to have two methods:
```
const RDKit::RWMol &get() const;
RDKit::RWMol &get();
```